generischedaten/SingleLinkedList/SingleLinkedList.cs

```diff
@@ -164,44 +183,21 @@ public void PrintList()
                 Console.WriteLine("The list is empty.");
             }
         }
-        void insertionSort(Node<T> headref)
-        {
-            sorted = null;
-            Node<T> current = headref;
-
-            while (current != null)
-            {
-
-                Node<T> next = current.next;
-
-                sortedInsert(current);
-
-                // Update current
-                current = next;
-            }
-            head = sorted;
-        }
 
-        void sortedInsert(Node<T> newnode)
+        public void SwitchNodes(Node firstNode, Node secondNode)
         {
-            /* Special case for the head end */
-            if (sorted == null || sorted.data >= newnode.data)
-            {
-                newnode.next = sorted;
-                sorted = newnode;
-            }
-            else
+            var List = head;
```

Comment:
List ist kein guter Name für diese Variable ... besser wäre z.B. current -> -0.5 in clean code